keyboards/vitamins_included/keymaps/default/keymap.c

```diff
@@ -112,25 +113,27 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 
 /* Adjust (Lower + Raise)
  * ,-----------------------------------------------------------------------------------.
- * |      |Qwerty|Colemk|Dvorak|      |      |      |      |      |      |      |      |
+ * |      | Reset|      |      |      |      |      |      |      |      |      |  Del |
  * |------+------+------+------+------+------+------+------+------+------+------+------|
- * |      |      |      |      |      |      |      |Qwerty|Colemk|Dvorak|      |      |
+ * |      |      |      |Aud on|Audoff|AGnorm|AGswap|Qwerty|Colemk|Dvorak|      |      |
  * |------+------+------+------+------+------+------+------+------+------+------+------|
  * |      |      |      |      |      |      |      |      |      |      |      |      |
  * |------+------+------+------+------+------+------+------+------+------+------+------|
- * |      |      |      |      |      |Audoff|Audon |      |      |      |      |RGBMOD|
+ * |      |      |      |      |      |      |      |      |      |      |      |RGBMOD|
  * `-----------------------------------------------------------------------------------'
  */
-[_ADJUST] =  LAYOUT_ortho_4x12(
-  _______, QWERTY,  COLEMAK, DVORAK,  _______, _______, _______, _______, _______, _______, _______, _______,
-  _______, _______, _______, _______, _______, _______, _______, QWERTY,  COLEMAK, DVORAK,  _______, _______,
+[_ADJUST] = LAYOUT_ortho_4x12(
+  _______, QK_BOOT, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_DEL,
+  _______, _______, _______, AU_ON,   AU_OFF,  AG_NORM, AG_SWAP, QWERTY,  COLEMAK, DVORAK,  _______, _______,
   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
-  _______, _______, _______, _______, _______, AU_OFF,  AU_ON,   _______, _______, _______, _______, RGB_MOD
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, RGB_MOD
 )
 
-
 };
 
+layer_state_t layer_state_set_user(layer_state_t state) {
+  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+}
```

Comment:
As above.
```suggestion
```

Reply:
Using `update_tri_layer_state` and removing keycodes (or using tri-layer from core), would break the `MO(_ADJUST)` on the base layer so this should be reverted.

Reply:
Huh, I hadn't noticed that. You are correct though, the `MO(_ADJUST)` does not work.

Reply:
Reverted.

Reply:
`update_tri_layer_state` in `layer_state_set_user` will break access to the adjust layer, except through tri layer.   Any layer code triggers the function, and if both raise and lower aren't enabled, it turns it off.  That includes `MO(_ADJUST)`

the tri layer keycodes will trigger the behavior too, but only when pressing/releasing those keys. `MO(_ADJUST)` will operate correctly in this case (and is why you'd want these over the layer state setup, in many cases). 

It's a bit of a confusing behavior if you're not familiar with it.